game/data/messages.xml

```diff
@@ -1,26 +1,41 @@
 <messages>
     <message>
-        <type>org.apollo.game.message.impl.ButtonMessage</type>
+        <type>org.apollo.game.message.impl.decode.IfActionMessage</type>
```

Comment:
These class names aren't descriptive. They need to have meaningful names without abbreviations.

Reply:
That's the actual name Jagex uses, I prefer it but I can see how it's not descriptive. I'll change it back.

Reply:
> That's the actual name Jagex use

This is a fairly tired argument. A lot of what Jagex done is garbage.

Reply:
I'm not disagreeing, I'm just explaining the decision.